content/javascript/concepts/dates/terms/getFullYear/getFullYear.md

```diff
@@ -0,0 +1,27 @@
+---
+Title: 'getFullYear()' 
+Description: 'Return the current full year' 
+Subjects: 
+  - 'Web Development'
+Tags: 
+  - 'Date'
+  - 'Method'
+CatalogContent: 
+  - 'paths/web-development'
+---
+
+The **getFullYear** method returns the current four digit year
+
+## Syntax
+
+```js
+myDate.getFullYear()
+```
+
+## Example
+
```

Comment:
Needs a sentence explaining what the example is doing.

Reply:
I have added this